eZ/Publish/API/Repository/Values/User/RoleCreateStruct.php

```diff
@@ -18,6 +18,13 @@
 abstract class RoleCreateStruct extends ValueObject
 {
     /**
+     * ID of a role.
+     *
+     * @var int
+     */
+    public $id;
```

Comment:
@lolautruche @bdunogier It is used when creating a role draft from an existing published role:
https://github.com/ezsystems/ezpublish-kernel/pull/1376/files#diff-c4d78f93737fdf4d5fd48c19e33b65c7R220